README.md

```diff
@@ -321,11 +324,19 @@ in   | cmd.meter_ext.get_report      | null       |                         | Re
 
 #### Interface props
 
-Name       | Value example | Description
------------|---------------|-------------
-`delta_t`  |               | time delta
-`prv_data` |               | previous meter reading
-`unit`     |               |
+Name         | Value example | Description
+-------------|---------------|-------------
+`delta_t`    |               | time delta
```

Comment:
Since we're fixing this, can we add a bit more descriptive info here. Maybe `Time (in seconds) since the last reading.` I'm not sure it's in seconds, so change to whatever is appropriate.